aimmdb/ingest/README.rst

```diff
@@ -0,0 +1,14 @@
+Ingestion steps
+---------------
+
+.dat file contains commented lines (denoted by #) with metadata, and columnated data beneath metadata.
+
+- for all comment lines, if they contain a colon, remove starting hash and split at colon to create key-value pairs
+- the only comment line without colon should be the header for columnated data
+- place key-value pairs in metadata dictionary
+- use data in .dat file to calculate mu_trans, mu_fluor, and mu_ref, i.e:
+.. math::
+    - mu_{trans} = -ln(it/i0)
```

Comment:
In math mode let's be really precise. Instead of $it$ use $I_t$ or something like this.

Reply:
Similarly instead of $ln$ use $\ln$.